src/fosslight_util/write_opossum.py

```diff
@@ -116,7 +131,7 @@ def get_externalAttribution_dict(self):
             dict[url] = self.url
             dict[packageType] = self.packageType
 
-        return dict
+        return {key: value for key, value in dict if value is not None}
```

Comment:
@nicarl 
It occurs the unpack error.
Please fix the dict.items() instead of dict.